distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -735,4 +738,38 @@ public void clearCatalog() {
     public void clearCache() {
         console.runCommand(CLEAR_CACHE);
     }
+
+    public ValidatableResponse executeOpenSearch(String format, String... query) {
```

Comment:
I agree with @emanns95. We should move those methods to the `OpenSearchTestCommons` class.